### ISSUE TITLE
Support .net standard 1.3 and .net46

### ISSUE THREAD

**Issue by Zyano** [AngleSharp/AngleSharp.Io#24]
# New Feature Proposal
Currently this doesn't support the same .NET standard versions as the normal AngleSharp package. 

## Description
It would be nice if the io package supported the same versions for .net framework and .net standard as the normal AngleSharp. The issue seems to have been discussed in #9 but from looking at the csproj it currently only support Windows NT and .net standard 2.0

## Background
Easier framework alignment with AngleSharp

**Comment by FlorianRappl** [comment]
Hi @Zyano we use here some stuff that cannot be used with .NET Standard 1.x. This is because we make extensive use of IO capabilities (hence the name).

Maybe you find compatibility packages for the used functionality, but out of the box this is intentional and one of the reasons why AngleSharp.Io is a dedicated package.

PRs welcome.

(The alignment could also be done by setting AngleSharp to require .NET Standard 2.0 - but we want that the core functionality can be used as broad as possible)

**Comment by Zyano** [comment]
Hi @FlorianRappl  I played around with a bit and ran into the same issue I'm guessing you found during development. It appears FileWebRequest and FtpWebRequest was never backported to 1.X which means those would require alternative implementations. 

The FtpWebRequest would not be very hard to get around as there are a ton of open source (MIT licensed) FTP implementations in C# that supports 1.X

The FileWebRequest is another story. I have very little experience with this one I would assume that cross OS could be a pain to handle in custom libraries which is could be why it wasn't ported to .net standard before 2.0

On a positive note I did manage to get net46 working with simply including some framework specific references.
